### PR TITLE
fix(docs): Bring minio chart instructions up to date

### PR DESCRIPTION
**Pull request by bei-re**
Current minio chart in stable is out of date and not maintained. 

Updated minio chart to use minio repo. Updated instructions to 
see generated credentials.

Checklist:

* [x] My organization is added to [USERS.md](https://github.com/argoproj/argo/blob/master/USERS.md).

### CODE REVIEW

File: USERS.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ Currently, the following organizations are **officially** using Argo Workflows:
 1. [Ant Group](https://www.antgroup.com/)
 1. [AppDirect](https://www.appdirect.com/)
 1. [BasisAI](https://basis-ai.com/)
+1. [BEI.RE](https://www.bei.re/)
```
Comment (alexec) [inline USERS.md:20]: how can we verify your relationship with this org please?

**Review by alexec — COMMENTED** [review]
Looks good so far.

**Review by alexec — CHANGES_REQUESTED** [review]
Requested info on relationship to org.

File: USERS.md (hunk 1 above)
Comment (bei-re) [reply USERS.md:20]: Screen Shot 2020-11-24 at 12 36 43 AM

**Review by alexec — APPROVED** [review]

**Pull request merged**